Title generator use bundle hooks

leftclickben opened the pull request.

leftclickben committed:
Change structure of title generator plugin.

+ Use bundle hooks instead of label hooks, this enables modification of labels
  even when the user isn't adding or editing a label.
+ Labels are being removed and then re-added as editLabel() does not seem to
  work as expected.

leftclickben committed:
Change structure of title generator plugin.

+ Use bundle hooks instead of label hooks, this enables modification of labels
  even when the user isn't adding or editing a label.
+ Labels are being removed and then re-added as editLabel() does not seem to
  work as expected.

leftclickben committed:
Remove error_log() accidentally committed.

leftclickben committed:
Remove error_log() accidentally committed.

leftclickben commented:
Chances are there is something I've missed here.  We need to do a dev box before we merge this one!

kehh commented:
I was going to ask for one ;) I'm off home now for fear of getting locked in. 
Cheers, 
K 

----- Original Message -----

From: "Ben New" notifications@github.com 
To: "wamuseum/providence" providence@noreply.github.com 
Sent: Monday, 19 May, 2014 5:33:27 PM 
Subject: Re: [providence] Title generator use bundle hooks (#32) 

Chances are there is something I've missed here. We need to do a dev box before we merge this one! 

— 
Reply to this email directly or view it on GitHub .

leftclickben committed:
Merge branch 'TitleGeneratorUseBundleHooks' of github.com:leftclickben/providence into TitleGeneratorUseBundleHooks

leftclickben committed:
Plugin enhancements.

+ Use correct hook method to generate title when object is saved, with relationships, as a new record
+ Only call update() on record being saved when a label has been changed, and remove previous loop prevention mechanism
+ Remove hardcoded locale id

leftclickben committed:
Change != to !==

leftclickben committed:
Restore bundle hooks.

+ Restore bundle hooks in addition to hookSaveItem so that titles are always updated regardless of method of saving data.
+ Code cleanup, remove daggy method separators.
